tests/general/InvalidLinkBearTest.py

```diff
@@ -396,20 +396,20 @@ def test_links_to_ignore(self):
         http://coalaisthebest.com/
         http://httpbin.org/status/404
         http://httpbin.org/status/410
-        http://httpbin.org/status/500
-        http://httpbin.org/status/503
+        http://httpbin.org/status/500/
```

Comment:
dont modify these lines.  Anything using `httpbin.org/status` is about testing error codes.

add new lines specifically for testing trailing `/` and any other improved matches your code changes allow.